src/locale/locales/zh-TW/messages.po

```diff
@@ -3220,11 +3162,11 @@ msgstr "至少有一張圖片缺失了替代文字。"
 
 #: src/screens/Onboarding/StepProfile/index.tsx:120
 msgid "Only .jpg and .png files are supported"
-msgstr ""
+msgstr "僅支援 .jpg 和 .png 文件"
 
 #: src/view/com/threadgate/WhoCanReply.tsx:100
 msgid "Only {0} can reply."
-msgstr "只有 {0} 可以回覆。"
+msgstr "只有{0}可以回覆。"
```

Comment:
```suggestion
msgstr "只有 {0} 可以回覆。"
```

Reply:
This should be with no spaces, lol, i forget and sunmit your suggestion